comit/src/expiries.rs

```diff
@@ -712,13 +716,17 @@ impl BobState {
         };
 
         match next_action {
+            Start => {
+                // This time is needed for Alice to act.
+                // When a swap in progress, this time is not taken in
+                // account to know whether Bob can still complete.
+                c.broadcast_alpha_fund_transaction()
+            }
             // Once Alice starts we need at least this much time, note that c.start() is not
             // included otherwise the actual time Alice takes invalidates the swap for Bob.
             WaitForAlphaFundTransactionFinality => {
                 // Transition from Started to AlphaFunded
-                c.broadcast_alpha_fund_transaction()
-                    + c.mine_alpha_fund_transaction()
-                    + c.finality_alpha()
```

Comment:
I don't agree with this change, can you explain please? Transitioning from Started to AlphaFunded includes the broadcast_aplha_fun_transaction period.